### PR TITLE
Added Blackjack Game wrt  #81

### LINKED ISSUE #81: Blackjack game
Want to add a blackjack game developed using Javascript, html and css.

### PR DESCRIPTION
**Pull request by TheArushiSingh**
added blackjack game wrt  #81 
developed using Javascript, HTML and CSS

Fix #81

### CODE REVIEW

File: Blackjack Game/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Blackjack game
+
```
Comment (harshita2216) [inline Blackjack Game/README.md:2]: Add a small description of the project please!

**Review by harshita2216 — CHANGES_REQUESTED** [review]
@TheArushiSingh, please add your project in the index.md file as well. 
Thanks!

**Review by harshita2216 — COMMENTED** [review]
Also, add your project to the index.md file.

File: Blackjack Game/README.md (hunk 1 above)
Comment (TheArushiSingh) [reply Blackjack Game/README.md:11]: done

**Comment by TheArushiSingh** [comment]
> Also, add your project to the index.md file.
Added.

**Comment by TheArushiSingh** [comment]
Added description in README.md file and I've also added the project to index.md file

**Pull request merged**